ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -101,6 +101,11 @@ export class EvalCommand extends Command<EvalCommandReturn> {
       const { context } = current;
       const { cfgu } = context;
 
+      if (!isStringInCfguType(current.context.schema)) {
+        throw new Error(
+          `Type '${current.context.schema}' is not yet supported in this SDK.\nFor the time being, please utilize the String type.\nWe'd greatly appreciate it if you could open an issue regarding this at https://github.com/configu/configu/issues/new/choose so we can address it in future updates.`,
+        );
+      }
```

Comment:
Better handled in `ConfigSchema.parse` rather than `EvalCommand`, since this would also apply to other commands like `UpsertCommand`.